src/mock_crust/support.rs

```diff
@@ -115,6 +115,21 @@ impl Network {
         let _ = imp.blocked_connections.remove(&(sender, receiver));
     }
 
+    /// Simulates the lost of connection.
+    pub fn lost_connection(&self,
+                           node_1: Endpoint,
+                           node_2: Endpoint,
+                           peer_1: PeerId,
+                           peer_2: PeerId) {
```

Comment:
It would be safe to extract the endpoint ID from the peer ID, wouldn't it? It seems that peer IDs are always constructed from endpoint IDs in mock_crust land: https://github.com/maidsafe/routing/blob/9db123e7b28acddb53e125376ae314a8b1f582fa/src/mock_crust/support.rs#L239